README.md

```diff
@@ -59,46 +59,46 @@ The [`config.yaml`](installer/config.yaml) file is structured to outline key com
 
 ```yaml
 ---
-rhtapCLI:
+cli:
   namespace: tssc
-  features: {}
+  products: {}
   dependencies: {}
 ```
 
-The attributes of the `rhtapCLI` object are as follows:
+The attributes of the `cli` object are as follows:
```

Comment:
The name `cli`, as a executable on `${PATH}` is too generic, we need a name like `tssc-cli`.

Reply:
Fixed